salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Setup MetalK8s sparse loop device %I
+RequiresMountsFor=/var/lib/metalk8s/storage/sparse
+
+[Service]
+Type=oneshot
+ExecStart=/sbin/losetup --find --partscan "/var/lib/metalk8s/storage/sparse/%i"
+ExecStop=/usr/local/bin/metalk8s-volume-clean-up "%i"
+TimeoutSec=60
```

Comment:
Actually not sure we want/need such timeout.

Reply:
OK, indeed it's almost an instantaneous operation.

Reply:
Removed.